clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/heartbeat/DefaultPushJobHeartbeatSenderFactory.java

```diff
@@ -37,41 +38,25 @@ public class DefaultPushJobHeartbeatSenderFactory implements PushJobHeartbeatSen
 
   @Override
   public PushJobHeartbeatSender createHeartbeatSender(
+      String kafkaUrl,
       VeniceProperties properties,
       @Nonnull ControllerClient controllerClient,
       Optional<Properties> sslProperties) {
     Validate.notNull(controllerClient);
     final String heartbeatStoreName = getHeartbeatStoreName(properties);
```

Comment:
Do you think it is worth controlling the store name via configs? Should we remove this?

Reply:
Discussed offline, use `AvroProtocolDefinition.BATCH_JOB_HEARTBEAT.getSystemStoreName()` instead.